src/ds/d400/d400-motion.cpp

```diff
@@ -80,10 +80,13 @@ namespace librealsense
         }
         catch (...) {}
 
+        double gyro_scale_factor = _fw_version >= firmware_version( 5, 15, 1, 224 ) ? 0.0001 : 0.1 ;
```

Comment:
Use a common factor for the two places using it in the file.
Can be a common member in the base class, initialized to 0.1 and set to 0.0001 in the base constructor if needed. Pass factor here and 1/factor in d400_motion. Document that FW factor was 10 up to FW version 5.16 and changed to 10000 to support higher gyro sensitivity.